roles/keystone/meta/main.yml

```diff
@@ -32,3 +32,5 @@ dependencies:
     vars:
       upload_helm_chart_src: "{{ keystone_helm_chart_path }}"
       upload_helm_chart_dest: "{{ keystone_helm_chart_ref }}"
+  - role: keycloak
+    when: atmosphere_keycloak_enabled | default(true)
```

Comment:
since keycloak is an "independent" thing for many services, i think it should be added to the playbook directly rather than a dependency for keystone.